lib/ramble/ramble/spack_runner.py

```diff
@@ -617,8 +617,8 @@ def get_package_path(self, package_spec):
             self._dry_run_print(self.spack, name_args)
             self._dry_run_print(self.spack, loc_args)
 
-            name = os.path.join(shlex.split(package_spec)[0])
-            location = os.path.join("dry-run", "path", "to", shlex.split(package_spec)[0])
+            name = shlex.split(package_spec)[0].split("@")[0]
```

Comment:
This change is independent of the main change of this PR. I noticed that currently the dry-run would insert this path in the variables as `{"maxtext@main": "/path/to..."}` such that referencing `{maxtext}` in `ramble.yaml` doesn't work as expected under dry-run.

Reply:
I feel like I recently fixed this as well, but it might be in the package manager stuff. So we might get a conflict with this when we merge that in but it should be easy to correct.